user_modules/IsReady_Template/isReady.lua

```diff
@@ -1360,7 +1360,7 @@ end
 			--Start SDL
 			Test["Precondition_StartSDL_" ..tostring(TestCaseName) ] = function(self)
 
-				StartSDL(config.pathToSDL, config.ExitOnCrash)
+				StartSDL(commonPreconditions:GetPathToSDL(), config.ExitOnCrash)
```

Comment:
@istoimenova, GetPathToSDL() function is not required as a parameter for StartSDL() since both versions (with or without "/" at the end will work)